packages/pools/src/utils/boosted/apr/getBoostedPoolApr.ts

```diff
@@ -0,0 +1,32 @@
+import { Address, PublicClient } from 'viem'
+import { ChainId } from '@pancakeswap/chains'
+import { checkIsBoostedPool } from '../checkIsBoostedPool'
+import { fetchAlpBoostedPoolApr } from './fetchAlpBoostedPoolApr'
+
+interface GetBoostedPoolApr {
+  client: PublicClient
+  sousId: number
+  contractAddress: Address
+  chainId: number | undefined
+}
+
+export const getBoostedPoolApr = async ({
+  client,
+  sousId,
+  contractAddress,
+  chainId,
+}: GetBoostedPoolApr): Promise<number> => {
+  const isBoostedPool = Boolean(chainId && checkIsBoostedPool(contractAddress, chainId))
+
+  if (!contractAddress || !chainId || !isBoostedPool) {
+    return 0
+  }
+
+  // Arbitrum ALP pools
+  if (chainId === ChainId.ARBITRUM_ONE && sousId === 2) {
```

Comment:
We shouldn't be using chain/pool identifier to tell what booster should be applied. The booster config should tell that itself. That's why I suggest we add a `BoosterType` to the booster config. Only when booster type is alp then we need to try get the alp apr

```suggestion
  if (boosterType === ALP) {
```

Reply:
// TODO